tests/faker.py

```diff
@@ -0,0 +1,24 @@
+"""
+This file holds reusable fake stuff for mocking the tests.
+"""
+
+FAKE_CHANNEL = 'C12942JF92'
```

Comment:
If we did that, wouldn't we have to remember all the places we used the`FAKE_CHANNEL` and then change those as well? I think we should go with @kola-er suggestion and avoid that technical debt.